test/CrowdsaleGenTest.js

```diff
@@ -106,7 +111,7 @@ contract('LifCrowdsale Property-based test', function(accounts) {
       (nextBlock > endBlock2) ||
       (state.crowdsalePaused) ||
       (state.crowdsaleFinalized) ||
-      (weiCost == 0);
+      (command.eth == 0);
```

Comment:
not really very related :)
Do you think I should do a separate commit, or even a separate PR?